### ISSUE TITLE
[Bug]: Relative paths required for referencing tom-select SCSS files

### ISSUE THREAD

**Issue by Kovah** [orchidjs/tom-select#809]
### Bug description

After upgrading to version 2.4.0, the following import no longer works:
```scss
@import "~tom-select/src/scss/tom-select.bootstrap5";
```

When running the build, I get this error:

```
ERROR in ./resources/assets/sass/app.scss
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
Can't find stylesheet to import.
  ╷
1 │ @import "~tom-select/src/scss/tom-select.bootstrap5";
  │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ╵
  resources/assets/sass/third-party/tom-select/_select.scss 1:9  @import
  resources/assets/sass/app.scss 8:9                             root stylesheet

```

This can only be fixed by referencing tom-select relative to the importing file like this:

```scss
@import "../../../../../node_modules/tom-select/src/scss/tom-select.bootstrap5";
```

A complete log is available in this pipeline of an open source project: https://github.com/Kovah/LinkAce/actions/runs/11912484409/job/33196168434

### Expected behavior

`@import "~tom-select/src/scss/tom-select.bootstrap5";` should work as expected and allow SCSS to import the file.

### Steps to reproduce

1. Download / check out LinkAce at commit 74e916bf: https://github.com/Kovah/LinkAce/tree/74e916bf1b02e32c9352449a50abb559e88cb8e3
2. npm install
3. npm run production

### Additional context

- Version 2.4.0

**Comment by mmarton** [comment]
Had a similar issue with select2, maybe relalated/helps:
https://github.com/apalfrey/select2-bootstrap-5-theme/issues/91

**Comment by nwalters512** [comment]
Thanks for the report! This will be resolved by #811. If you're looking for an immediate fix, you can update the import path to `~tom-select/dist/scss/tom-select.bootstrap5`. I'd recommend making this change anyways, as we'll likely stop shipping `src` in the next major version.
